color_refinement.py

```diff
@@ -148,21 +148,26 @@ def fast_color_refine(graph: Graph, coloring: Coloring) -> Coloring:
                 split_count += 1
                 # Each color_class is added to the list
                 new_color_classes.append(new_color)
-            # The smallest_color here is the first color added to the list, which is the original color
-            smallest_color = new_color_classes[0]
+            # The largest_color here is the first color added to the list, which is the original color
+            largest_color = new_color_classes[0]
             if split_count > 1:
                 debug('New color classes:', new_color_classes)
                 # If the original color is in the queue, all the other colors should be added to the queue
-                if qlist.find(smallest_color) is not None:
+                if qlist.find(largest_color) is not None:
```

Comment:
Vervang deze door 'color_class', dan is het duidelijk dat we kijken naar de originele kleur.

Reply:
Ik snap niet wat je hier bedoeld.

Reply:
` If the original color is in the queue` dus dan is het netter om hier `if qlist.find(color_class) is not None:` te testen ipv `if qlist.find(largest_color) is not None:`

Reply:
Ik zou nog steeds graag `if qlist.find(color_class) is not None:` zien in regel 156